Packs/WindowsDefenderAdvancedThreatProtection/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "Windows Defender Advanced Threat Protection (Deprecated)",
     "description": "Deprecated. Use the Microsoft Defender for Endpoint pack instead.",
     "support": "xsoar",
-    "hudden": true,
+    "hidden": true,
```

Comment:
let's change the one item with `"fromversion"` (lowercase) to `fromVersion` either in this same PR or in another, your choice

Reply:
Same PR